boards/arm/cc3220sf_launchxl/doc/cc3220sf_launchxl.rst

```diff
@@ -248,6 +247,29 @@ Once the connection succeeds, the network co-processor keeps the AP identity in
 its persistent memory.  Newly loaded WiFi applications then need not explicitly
 execute any WiFi scan or connect operations, until the need to change to a new AP.
 
+Secure Socket Offload
+*********************
+
+The SimpleLink WiFi driver provides socket operations to the Zephyr socket
+offload point, enabling Zephyr BSD socket API calls to be directed to the
+SimpleLink WiFi driver, by setting :option:`CONFIG_NET_SOCKETS_OFFLOAD`
+to ``y``.
+
+Secure socket (TLS) communication is handled as part of the socket APIs, and
```

Comment:
Since this is a long sentence with multiple items to do, it would read better as a bullet list:
```
Secure socket (TLS) communication is handled as part of the socket APIs, 
and enabled by:

- setting both :option:`CONFIG_NET_SOCKETS_SOCKOPT_TLS` and
  :option:`CONFIG_TLS_CREDENTIAL_FILENAMES` to ``y``,
- using the TI Uniflash tool to program the required certificates and
  keys to the secure flash filesystem, and enabling the TI Trusted Root-Certificate Catalog.
```

Reply:
OK, updated per request.  Thanks.